sdk/src/test/scala/com/horizen/account/fixtures/ForgerAccountFixture.scala

```diff
@@ -35,10 +34,8 @@ object ForgerAccountFixture extends SecretFixture {
     }
     val blockSignProposition = signerPrivKey.publicImage()
 
-
-    // create private/public key pair
-    val pair: ECKeyPair = Keys.createEcKeyPair
-    val ownerAddressProposition = new AddressProposition(BytesUtils.fromHexString(Keys.getAddress(pair)))
+    // TODO get a deterministic value with createEcKeyPair
+    val ownerAddressProposition = new AddressProposition(util.Arrays.copyOf(byteSeed, Account.ADDRESS_SIZE))
```

Comment:
we may use `Secp256k1.createKeyPair` method. it is deterministic.

Reply:
Let's discuss it, I have a doubt. This method internally makes use of `SecureRandom(seed)`